man/wacom.man

```diff
@@ -26,9 +28,6 @@ and requires the wacom kernel driver being loaded before this driver starts.
 Please check https://github.com/linuxwacom for latest updates of Wacom X
 and kernel drivers.
 .SH DRIVER-INTERNAL DEVICE HOTPLUGGING
-When input device hotplugging in the X server is enabled and no
-.B InputDevice
-section exists for a compatible tablet device and an
 .B InputClass
 section (see xorg.conf.d(5x)) assigns this driver for the device, the
```

Comment:
not related to your PR but while you're editing this anyway: iirc the `5x` is compression related and up to the distribution, this should just be:

```suggestion
section (see xorg.conf.d(5)) assigns this driver for the device, the
```

Reply:
I found other "5x"s in the description and replaced them to "5" on the commit.